openapi.yml

```diff
@@ -409,7 +409,35 @@ paths:
               schema:
                 type: string
                 example: Switch not found
+  /v3/interfaces/tag_ranges:
+    get:
+      summary: Get tag_ranges and available_tags from all interface
+      responses:
+        '200':
+          description: Ok
+          content:
+            application/json:
+              schema:
+                type: object
```

Comment:
I'll recommend that you further specify the object schema just so front-end/clients can rely on the spec to know the shape of the response object.

Reply:
Specified more of the returned object and added an example. Looks like openapi 3.0.0 does not support regex in properties so just wrote `switch_id`, `interface_index`, and `interface_id` instead.